include/openssl/ssl3.h

```diff
@@ -256,6 +259,7 @@ extern "C" {
  */
 # define SSL3_CT_NUMBER                  9
 
+/* No longer used as of OpenSSL 1.1.1 */
 # define SSL3_FLAGS_NO_RENEGOTIATE_CIPHERS       0x0001
```

Comment:
Yes, there is potential for binary compatibility issues if the define value gets changed.